lib/lz4.c

```diff
@@ -1727,7 +1727,16 @@ LZ4_decompress_generic(
 
                 /* Fastpath check: Avoids a branch in LZ4_wildCopy32 if true */
                 if (!(dict == usingExtDict) || (match >= lowPrefix)) {
+#if defined(__aarch64__) && !defined(__clang__)
```

Comment:
If you want your patch to only be triggered for `gcc`, you might want to test `__GNUC__` too.
This would filter out other non-clang compilers, such as Visual for example.

Reply:
OK, I will check this part.